internal/bminventory/inventory.go

```diff
@@ -3637,18 +3636,32 @@ func (b *bareMetalInventory) changeDNSRecordSets(ctx context.Context, cluster co
 			dnsRecordSetFunc = dnsProvider.DeleteRecordSet
 		}
 
+		apiVip := cluster.APIVip
+		ingressVip := cluster.IngressVip
+		if swag.StringValue(cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
+			apiVip, _ = common.GetBootstrapMachineNetworkAndIp(&cluster)
+			ingressVip = apiVip
+			// Create/Delete A record for API-INT virtual IP
+			_, err := dnsRecordSetFunc(domain.APIINTDomainName, apiVip)
+			if err != nil {
+				log.WithError(err).Errorf("failed to update DNS record: (%s, %s)",
+					domain.APIINTDomainName, apiVip)
+				return err
+			}
```

Comment:
Why have this?
The same logic exists in line 3654.

Reply:
Not the same. On 3654 we set api. Here api-int